.cicd/docker/amazonlinux-2.dockerfile

```diff
@@ -72,11 +72,23 @@ ENV PATH=${PATH}:/mongodb-linux-x86_64-amazon-3.6.3/bin
 RUN curl -LO http://download-ib01.fedoraproject.org/pub/epel/7/x86_64/Packages/c/ccache-3.3.4-1.el7.x86_64.rpm \
   && yum install -y ccache-3.3.4-1.el7.x86_64.rpm
 
+# Install Buildkite Agent
+RUN echo -e "[buildkite-agent]\nname = Buildkite Pty Ltd\nbaseurl = https://yum.buildkite.com/buildkite-agent/stable/x86_64/\nenabled=1\ngpgcheck=0\npriority=1" > /etc/yum.repos.d/buildkite-agent.repo && \
+  yum -y install buildkite-agent
+
 # PRE_COMMANDS: Executed pre-cmake
 # CMAKE_EXTRAS: Executed right before the cmake path (on the end)
-ENV PRE_COMMANDS="export PATH=/usr/lib64/ccache:$PATH &&"
+ENV PRE_COMMANDS="export PATH=/usr/lib64/ccache:\$PATH"
 ENV CMAKE_EXTRAS="-DCMAKE_CXX_COMPILER='clang++' -DCMAKE_C_COMPILER='clang'"
 
-CMD bash -c "$PRE_COMMANDS ccache -s && \
-    mkdir /workdir/build && cd /workdir/build && cmake -DCMAKE_BUILD_TYPE='Release' -DCORE_SYMBOL_NAME='SYS' -DOPENSSL_ROOT_DIR='/usr/include/openssl' -DBUILD_MONGO_DB_PLUGIN=true $CMAKE_EXTRAS /workdir && make -j $(getconf _NPROCESSORS_ONLN) && \
-    ctest -j$(getconf _NPROCESSORS_ONLN) -LE _tests --output-on-failure -T Test"
\ No newline at end of file
+# Bring in helpers that provides execute function so we can get better logging in BK and TRAV
+COPY ./docker/.helpers-v33 /tmp/.helpers
```

Comment:
Do we need multiple helper scripts?
Can they be named more clearly?

Reply:
Agreed, changing the name (I forgot to do this before the PR)